client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,197 @@
-		const comp = shallow( <Banner { ...props } callToAction={ 'Upgrade Now!' } /> );
-		expect( comp.find( '.has-call-to-action' ) ).toHaveLength( 1 );
+		const { container } = render( <Banner { ...props } callToAction={ 'Upgrade Now!' } /> );
+		expect( container.firstChild ).toHaveClass( 'has-call-to-action' );
 	} );
 
 	test( 'should not have .has-call-to-action class if callToAction is null', () => {
-		const comp = shallow( <Banner { ...props } callToAction={ null } /> );
-		expect( comp.find( '.has-call-to-action' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } callToAction={ null } /> );
+		expect( container.firstChild ).not.toHaveClass( 'has-call-to-action' );
 	} );
 
 	test( 'should render a <Button /> when callToAction is specified', () => {
-		const comp = shallow( <Banner { ...props } callToAction={ 'Buy something!' } /> );
-		expect( comp.find( Button ) ).toHaveLength( 1 );
+		render( <Banner { ...props } callToAction={ 'Buy something!' } /> );
+		expect( screen.getByRole( 'button' ) ).toHaveTextContent( 'Buy something!' );
 	} );
 
 	test( 'should not render a <Button /> when callToAction is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( Button ) ).toHaveLength( 0 );
+		render( <Banner { ...props } /> );
+		expect( screen.queryByRole( 'button' ) ).toBeNull();
 	} );
 
 	test( 'should have .is-jetpack class and JetpackLogo if jetpack prop is defined', () => {
 		const { plan, ...propsWithoutPlan } = props;
-		const comp = shallow( <Banner { ...propsWithoutPlan } jetpack /> );
-		expect( comp.find( '.is-jetpack' ) ).toHaveLength( 1 );
-		expect( comp.find( 'JetpackLogo' ) ).toHaveLength( 1 );
+		const { container } = render( <Banner { ...propsWithoutPlan } jetpack /> );
+		expect( container.firstChild ).toHaveClass( 'is-jetpack' );
+		expect( container.getElementsByTagName( 'svg' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should render have .is-horizontal class if horizontal prop is defined', () => {
-		const comp = shallow( <Banner { ...props } horizontal /> );
-		expect( comp.find( '.is-horizontal' ) ).toHaveLength( 1 );
+		const { container } = render( <Banner { ...props } horizontal /> );
+		expect( container.getElementsByClassName( 'is-horizontal' ) ).toHaveLength( 1 );
 	} );
 
 	test( 'should render a <PlanPrice /> when price is specified', () => {
-		const comp = shallow( <Banner { ...props } price={ 100 } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 1 );
+		render( <Banner { ...props } price={ 100 } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
 	} );
 
 	test( 'should render two <PlanPrice /> components when there are two prices', () => {
-		const comp = shallow( <Banner { ...props } price={ [ 100, 80 ] } /> );
-		expect( comp.find( PlanPrice ) ).toHaveLength( 2 );
+		render( <Banner { ...props } price={ [ 100, 80 ] } /> );
+		expect( screen.getByText( '100' ) ).toBeVisible();
```

Comment:
Same as above, how can we be sure if these are displayed as prices and not as just general text?